modules/openapi-generator/src/main/resources/htmlDocs2/index.mustache

```diff
@@ -452,6 +452,10 @@
                                         }
                                         if (schema.$ref != null) {
                                           schema = defsParser.$refs.get(schema.$ref);
+                                          Object.keys(schema.properties).forEach( (item) => {
+                                            if (schema.properties[item].$ref != null) {
+                                              schema.properties[item] = defsParser.$refs.get(schema.properties[item].$ref);
+                                            }
```

Comment:
@wing328 The forEach-Loop is never closed (missing `});`). I opened this issue https://github.com/OpenAPITools/openapi-generator/issues/12842

Reply:
@kas1337 thanks for catching the issue. Can you please file a PR with the suggested fix?